modules/core/core-api/src/main/java/com/enonic/xp/content/ContentConstants.java

```diff
@@ -41,9 +45,7 @@
         value( "master" ).
         build();
 
-    public static final String CONTENT_REPO_ID_PREFIX = "com.enonic.cms.";
```

Comment:
this breaks backwards compatibility (API compilation). We have to preserve it, but deprecate